src/pages/ProductList/components/ProductList/Product/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+import urls from '../../../../../constants/urls'
+import { Wrapper, ImgWrap, Img, TitleWrap, Title, Price, Link } from './styled'
+
+const Product = ({ node }) => (
+  <Wrapper>
```

Comment:
Overflowing Product on ProductList

Reply:
<img width="320" src="https://user-images.githubusercontent.com/12123231/55395427-d5389080-5541-11e9-9f30-1d4cb34740c7.png" />

Reply:
Fixed to 100% of width.